tldw_Server_API/app/main.py

```
@@ -5390,6 +5390,18 @@ def _should_run_late_stop(task_name: str, task: Any) -> bool:
 
+    # Sweep any remaining managed backends that were created outside the media
+    # DB dependency cache (for example direct managed_media_database callers).
+    try:
+        from tldw_Server_API.app.core.DB_Management.backends.factory import (
+            close_all_backends as _close_all_backends,
+        )
+
+        _close_all_backends()
+        logger.info("App Shutdown: Database backend registries cleared")
+    except _STARTUP_GUARD_EXCEPTIONS as e:
+        logger.debug(f"App Shutdown: Database backend registry cleanup skipped/failed: {e}")
```

Ruff (0.15.11)

[warning] Logging statement uses f-string

(G004)